cdap-watchdog/src/main/java/io/cdap/cdap/logging/appender/system/LogFileManager.java

```diff
@@ -43,7 +43,9 @@
 /**
  * Class including logic for getting log file to write to. Used by {@link CDAPLogAppender}
  */
-final class LogFileManager implements Flushable, Syncable {
+public final class LogFileManager implements Flushable, Syncable {
+  public static final String FOLDER_DATE_FORMAT = "yyyy-MM-dd";
```

Comment:
This class should be using this constant too? Also, instead of exposing a static string for the format, it is better to expose it as a method, something like `getLogDirectory(long timestamp)`.

Reply:
Created a function for `String formatLogDirectoryName(long timestamp)`.